src/core/settings.py

```diff
@@ -110,7 +110,7 @@ class Config:
 
     # ## Mail settings #
 
-    TECH_EMAIL = getenv('TECH_EMAIL', 'tech@ubiquitypress.com')
+    TECH_EMAIL = environ['TECH_EMAIL']
```

Comment:
Shall we try to keep this consistent and use either `environ` or `getenv` everywhere in the same file?

Reply:
I have tried to introduce `environ` to indicate values that are required, so I guess I can change the `getenv` calls to `environ.get` :+1: